src/pages/unsere.html

```diff
@@ -0,0 +1,190 @@
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Rest APIs
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Redux
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          test driven development
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Data modeling
+        </li>
+      </ul>
+      <ul class="flex flex-wrap gap-4 text-sm">
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          JWT
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          CSS
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          frontend architecture
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Figma
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Nest.JS
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          ORM
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Prisma
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          GraphQL
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          CI / CD
+        </li>
+        <li
+          class="text-[#001342] border bg-white rounded-full p-3 font-raleway font-semibold flex items-center justify-center">
+          Scum Framework
+        </li>
+      </ul>
+    </div>
+    <div>
+      <button class="text-green-primary font-Roboto text-xs font-semibold">
+        ZUM TECH-SKILLPROFIL
+      </button>
+    </div>
+    <div id="ojembees" class="flex flex-col items-end">
+      <div class="font-bold text-lg text-white w-full relative z-10">
+        Eine mögliche Teamkonstellation
+        <div
+          class="h-1 bg-green-primary-trans w-44 absolute top-[18px] lg:left-36 -rotate-2"></div>
```

Comment:
bg-green-primary-trans is not descriptive,check the following link [https://classic-cobalt-104.notion.site/CSS-color-naming-format-09b4d1a0d92e435aa2384753514b91c3](url)